hooks/addon

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 set -eu
 vault="secret/$GENESIS_VAULT_PREFIX"
-
-ip=$(bosh vms --json | jq -r '.Tables[0].Rows[0].ips')
+declare -a ips
+ips( =$(bosh vms --json | jq -r '.Tables[0].Rows[0].ips') )
```

Comment:
Switching to `ips=( $(bosh vms --json | jq -r '.Tables[0].Rows[0].ips') )` will fix addon